_content/papers/the-celestial-coral-tree-and-the-noble-disciple_dhammadinna.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "The Celestial Coral Tree and the Noble Disciple: Ekottarika-āgama Discourse 39.2"
+authors:
+  - dhammadinna
+external_url: "https://www.academia.edu/44794503/The_Celestial_Coral_Tree_and_the_Noble_Disciple_Ekottarika_%C4%81gama_Discourse_39_2"
+drive_links:
+  - "https://drive.google.com/file/d/1WDPH3ZBL2i7712DYQb1FSJtR_WfeMphO/view?usp=sharing"
+course: samadhi
+tags:
+  - EA
+  - renunciation
+  - path
+year: 2021
+publisher: "Nuvis"
+address: "Paris"
+booktitle: "Dharmayātrā, Felicitation Volume in Honour of Ven. Dr. Tampalawela Dhammaratana"
+pages: "92--99"
+---
+
```

Comment:
Let's add another paragraph here with a quote from the sutta:

```md

> ... the fourth meditative absorption, this is just like that tree gradually
 blooming.

```

Perhaps?

(Note the blank lines above and below the quote!)